src/dex/solidly/solidly.ts

```diff
@@ -335,10 +345,14 @@ export class Dystopia extends UniswapV2 {
     count: number,
   ): Promise<PoolLiquidity[]> {
     if (!this.subgraphURL) return [];
+
+    const stableFieldKey =
+      this.dexKey.toLowerCase() === 'solidly' ? 'stable' : 'isStable';
+
     const query = `query ($token: Bytes!, $count: Int) {
```

Comment:
For `SpiritSwapV2` there is a difference: `$token: String`, not `Bytes!`. Better to check with tests other forks if there subgraph query compatible or not

Reply:
That's unfortunate. Thx I will  check

Reply:
as discussed IRL (reporting for posterity):
1/ it looks like `Bytes` and 'String' are interchangeable when subgraph is expecting a string.
2/ [this](https://github.com/paraswap/paraswap-dex-lib/pull/167/commits/08f6f610e58e7ddf4bbcb14fbec6b28e7e27893c) was missing